tests/lambda_functions/va_profile/test_va_profile_integration.py

```diff
@@ -673,68 +790,47 @@ def test_va_profile_opt_in_out_lambda_handler_integration_testing(
             VAProfileLocalCache.communication_channel_id == 1,
         )
     )
-
     assert notify_db_session.session.scalar(stmt) == 0
 
+    # Create the event with appropriate parameters for COMP and PEN Opt-In
     event = create_event('txAuditId', 'txAuditId', '2022-04-07T19:37:59.320Z', va_profile_id, 1, 5, True, jwt_encoded)
-    event['queryStringParameters'] = {'integration_test': "the value doesn't matter"}
+
+    # Call the Lambda Handler with the test event
     response = va_profile_opt_in_out_lambda_handler(event, None)
 
+    # Validate the response from the lambda handler
     assert isinstance(response, dict)
     assert response['statusCode'] == 200
-    assert response.get('headers', {}).get('Content-Type', '') == 'application/json'
-    response_body = loads(response.get('body', '{}'))
-    assert 'put_body' in response_body
 
+    # Assert PUT request to VAProfile was made with correct parameters
     expected_put_body = {
         'dateTime': '2022-04-07T19:37:59.320Z',
         'status': 'COMPLETED_SUCCESS',
     }
+    mock_put_instance.request_assert_called_once_with('txAuditId', expected_put_body)
+    mock_put_instance.request.assert_called_once()
+
+    # Assert POST request to VANotify was made with correct parameters, including the expected month
+    mock_post_instance.request.assert_called_once_with(
+        'POST',
+        '/v2/notifications/sms',
+        body=json.dumps(
+            {
+                'template_id': 'mock_template_id',
+                'recipient_identifier': {'id_type': 'VAPROFILEID', 'id_value': str(va_profile_id)},
+                'sms_sender_id': 'mock_sms_sender_id',
+                'personalisation': {'month-name': expected_month},
+            }
+        ),
+        headers={'Authorization': f'Bearer {encoded_header}', 'Content-Type': 'application/json'},
+    )
 
-    put_mock.assert_called_once_with('txAuditId', expected_put_body)
```

Comment:
Ok - so I removed this because the **return value** of the lambda is different based on if the `intergration_test` flag is passed into the context. 

If the `intergration _test` is passed it - it returns the expected PUT request to VAProfile. If not, it only returns a 200.